app/routes/_index.tsx

```diff
@@ -1,19 +1,26 @@
-// import * as Switch from "@radix-ui/react-switch";
+import * as Switch from "@radix-ui/react-switch";
 import type { V2_MetaFunction } from "@remix-run/node";
 import { Link } from "@remix-run/react";
+import { Header } from "../components/header";
```

Comment:
utilizemos siempre absolute imports en vez de relative imports

```suggestion
import { Header } from "~/components/header";
```